src/nativeMain/kotlin/util/Util.kt

```diff
@@ -31,4 +32,18 @@ fun Display.sendClientMessage(type: Atom, window: Window, propagate: Boolean = f
 }
 
 fun Display.sendClientMessage(type: Atom, propagate: Boolean = false, eventMask: Long = NoEventMask, vararg data: Long) =
-    sendClientMessage(type, rootWindow, propagate, eventMask, *data)
\ No newline at end of file
+    sendClientMessage(type, rootWindow, propagate, eventMask, *data)
+
+public fun Window.setCursor(dpy: Display, cur: UInt = 2.toUInt()) { 
```

Comment:
`public` access modifier isn't necessary here, would look cleaner without.
`2.toUInt()` can be replaced with `2u`